src/ipwndfu/main.py

```diff
@@ -533,12 +537,15 @@ def repair_heap(device=None, match_device=None):
         heap_state = pwned.platform.heap_state
         heap_write_hash = pwned.platform.heap_write_hash
         heap_check_all = pwned.platform.heap_check_all
-        if (
-            heap_base == 0
-            or heap_offset == 0
-            or heap_state == 0
-            or heap_write_hash == 0
-            or heap_check_all == 0
+        if any(
+            _ == 0
```

Comment:
same critique - `_` is used for unused parameters right?  so `addr` or `value` is a betters name?  @nyuszika7h for style guidance

Reply:
Yeah, probably best not to use `_`. I usually just use one-letter variable names like `x` for comprehensions but `addr` would probably be good too.